examples/question-answering/README.md

```diff
@@ -145,7 +145,28 @@ Here is a DeepSpeed configuration you can use to train your models on Gaudi:
 ```
 
 
-### Recommended Hyperparameters for Mixed Precision
+## Inference
+
+To run only inference, you can start from the commands above and you just have to remove the training-only arguments such as `--do_train`, `--per_device_train_batch_size`, `--num_train_epochs`, etc...
+
+For instance, you can run inference with BERT on SQuAD on 1 Gaudi card with the following command:
+```bash
+python run_qa.py \
+  --model_name_or_path bert-large-uncased-whole-word-masking \
```

Comment:
should the model_name_or_path be the checkpoint?

Reply:
It can be a name of a model on the HF Hub, or the name of the folder where the checkpoint is stored.